linera-sdk/src/base/mod.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Module for the base runtime.
+
+mod conversions_from_contract_wit;
+mod conversions_from_service_wit;
+mod conversions_to_contract_wit;
+mod conversions_to_service_wit;
```

Comment:
It might be possible to join the contract/service module pairs so that we only have `conversions_from_wit` and `conversions_to_wit`. We could get the common code and place it in a macro, and then use `wit_base_api` identifier as the macro's parameter. Then it could be called like:

```rust
conversions_to_wit(crate::contract::wit::base_runtime_api);
conversions_to_wit(crate::service::wit::base_runtime_api);
```